### ISSUE TITLE
Just display part of it when using on the next.js instace and there is a scaling on Windows

### ISSUE THREAD

**Issue by AdairLi2504** [mosch/react-avatar-editor#402]
It just display part of the component when I use it on a next.js instace and the "Change the size of text, apps, and other items" on "Scale and layout" of "Display" is larger than 100%.

I use this code to test.
```javascript
import AvatarEditor from 'react-avatar-editor'

function Home() {
  

  return (
    <main>
      <AvatarEditor
        image="https://cdn.pixabay.com/photo/2023/10/13/14/39/book-8312948_1280.jpg"
        width={250}
        height={250}
        border={50}
        color={[255, 255, 255, 0.6]} // RGBA
        scale={1.2}
        rotate={0}
      />
    </main>
  )
}

export default Home
```
(App.js.The rest part of instace follow the template)
![image](https://github.com/mosch/react-avatar-editor/assets/71171259/da9f6dfc-5221-462e-ba8a-2311c82b108e)
(Github Codespace,use Next.js template)
![image](https://github.com/mosch/react-avatar-editor/assets/71171259/42a2a1d1-fdac-429e-9fd9-e659f0967380)
(Github Codespace,use ordinary React template)

**Comment by mosch** [comment]
Thank you for reporting this, however I'm not sure what the actual problem is. Can you rephrase?

**Comment by flyingsnoopy** [comment]
Sorry，I‘ve forgotten mention to that this bug seem that only shows on Windows.As the picture show, when I use it on a nextjs (v13.5.4) instance and the client setting  "Change the size of text, apps, and other items" on "Scale and layout" of "Display" on Windows Setting is larger than 100%,it only show the top and left part and its right and bottom border are cut off.[More information about Windows system scaling](pureinfotech.com/change-scale-layout-windows-10/)
(The red marks the "Change the size of text, apps, and other items" on "Scale and layout" of "Display")
![image](https://github.com/mosch/react-avatar-editor/assets/71171259/d8914744-5d5f-47b6-a0db-e1700e74c7a4)

**Comment by flyingsnoopy** [comment]
I'm so sorry that I'm not good at English😥
